### DISCUSSION TITLE
mysql limit

### DISCUSSION THREAD

**Discussion by EducatedMonkey33** [apache/netbeans#4896] [Q&A]
when I use the database feature it always sets the limit to 100 records and thus when i get the reply I have to go back to the top and increase the limit so i get all the requested records, I have tried to find were I can change this limit I seem to recall on the older version I was using before migrating to this new 64 bit I simply change it once and it remembered it but no such luck on version 15so please tell me how to change the limit as it annoying to keep doing things over

**Comment by 6PATyCb** [comment]


**Reply by 6PATyCb** [reply]
Oh I misunderstood you on my previous post. 
You speaking about it:
![image](https://user-images.githubusercontent.com/33724393/199261614-0528454e-2e39-48a2-abe4-4dc3b9925c47.png)

At NetBeans 15 on my desktop it is worked as you sad. Change it once and it remembered.
